Update application-gateway-faq.md

a-ketsuc opened the pull request:
Update the answer of  Q. How does Application Gateway support high availability and scalability?
After update : if you have 2 or more instances deployed.
Before update : if you have more than 2 instances deployed.
Old one makes customers get confused and since it's also related to SLA, this sentence should be more specific.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

a-ketsuc committed:
Update application-gateway-faq.md

Update the answer of  Q. How does Application Gateway support high availability and scalability?
After update : if you have 2 or more instances deployed.
Before update : if you have more than 2 instances deployed.
